cbits/encrypted_sign.c

```diff
@@ -253,9 +253,9 @@ static void serialize_index32(uint8_t *out, uint32_t index, derivation_scheme_mo
 
 static void add_left(ed25519_secret_key res_key, uint8_t *z, ed25519_secret_key priv_key, derivation_scheme_mode mode)
 {
-	uint8_t zl8[32];
+	ed25519_secret_key zl8;
 
-	memset(zl8, 0, 32);
+	memset(zl8, 0, 64);
```

Comment:
Because the size of the `z18` array has increased to `64` we set all `64` elements to zero.